dev/com.ibm.ws.session.cache_fat/publish/servers/sessionCacheServerB/server.xml

```diff
@@ -16,14 +16,39 @@
                   httpPort="${bvt.prop.HTTP_secondary}"
                   httpsPort="${bvt.prop.HTTP_secondary.secure}"/>
     
-    <httpSession maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="10m"/>
+    <httpSession maxInMemorySessionCount="1" allowOverflow="false" invalidationTimeout="10m">
+      <!-- Use app validation token to make sure we don't accidentally join up with other machines running this test at the same time -->
+      <properties hazelcast.application.validation.token="${hzl.cluster.id}"/>
```

Comment:
This should be set on a httpSessionCache element rather than the httpSession element.

Reply:
Also, same as above, can the hazelcast.application.validation.token be specified as a property this way?